remsfal-service/src/main/java/de/remsfal/service/control/ZeebeController.java

```diff
@@ -0,0 +1,64 @@
+package de.remsfal.service.control;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.Response;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.jboss.logging.Logger;
+
+@Path("/zeebe")
+public class ZeebeController {
+
+        @Inject
+        ZeebeClient zeebeClient;
+
+        @Inject
+        Logger logger;
+
+        @POST
+        @Path("/start")
+        public Response startWorkflow(Map<String, Object> requestBody) {
```

Comment:
It would be much more readable if the json body was implemented as a java class, e.g. remsfal-core/src/main/java/de/remsfal/core/json/**/*Json.java